bin/test_matchpycpp_gen_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
You have to put `set -ex` here. The `-e` is to fail the bashscript when one of the commands fail, and the `-x` is to show the commands, so that one can see in the Travis log what was happening (if something fails).

Reply:
Fixed in 7d3cbf5a62f3fe9faa97bd83b8f6c1d65c4315bf.